ui/src/common/components/EmptyOrChildren.jsx

```diff
@@ -28,8 +31,10 @@ EmptyOrChildren.propTypes = {
     PropTypes.object,
     PropTypes.instanceOf(List),
     PropTypes.instanceOf(Map),
+    PropTypes.number,
   ]),
   description: PropTypes.string,
+  extraInfo: PropTypes.node,
```

Comment:
why not just use `description`?

Reply:
`description` is a `<p>` in ant's `<Empty>`, so you cannot put something like 
```html
<div>0 results</div>
<div>Oops! Your might want to check out our 
  <ExternalLink ... >search tips</ExternalLink>
</div>
```
inside it